ftw/linkchecker/command/broken_link.py

```diff
@@ -0,0 +1,73 @@
+from plone import api
+from plone.dexterity.utils import safe_utf8
+
+
+class BrokenLink(object):
+    table_attrs = [
+        'is_internal',
+        'link_origin',
+        'link_target',
+        'status_code',
+        'content_type',
+        'response_time',
+        'header_location',
+        'error_message',
+    ]
+    counter = 0
+
+    def __init__(self):
+        self.is_broken = None
+        self.is_internal = None
+        self.link_origin = 'Unknown origin'
+        self.link_target = 'Unknown target'
+        self.status_code = 'Unknown status code'
+        self.content_type = 'Unknown content type'
+        self.response_time = 'Unknown response time'
+        self.header_location = 'Unknown header location'
+        self.error_message = 'No error occurred'
+        self.counter = BrokenLink.counter
+        BrokenLink.counter += 1
```

Comment:
Do you need this for anything?
It's still not a good pattern to manipulate or use the class variable like that.